addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -66,10 +66,23 @@ def _l10n_it_edi_check_invoice_configuration(self, invoice):
 
         return errors
 
+    def _l10n_it_edi_is_reverse_charge(self, invoice):
+        company_country = invoice.company_id.country_id
+        partner_country = invoice.commercial_partner_id.country_id
+        return (
+            invoice.is_outbound()
```

Comment:
should not that be in_bound?

Reply:
We are considering the case in which we have a vendor bill, that normally we ain't able to export it.
Outbound means in_invoice and out_refund
![immagine](https://user-images.githubusercontent.com/1665365/173329556-f64ac13c-dab3-4268-95b4-774c6c43d0c9.png)

Reply:
"outbound" is about the payment then or something...  is_purchase_document would be more logical.  Because what happens in case of a reverse charge credit note?

Reply:
Good question. What should happen? Probably should be exported as well

Reply:
Changing it. We'll have to see the case anyway.